basic/barrier/src/main/resources/META-INF/spring/integration/client-context.xml

```diff
@@ -14,8 +14,14 @@
 	<int:channel id="requestChannel"/>
 
 	<int-http:outbound-gateway request-channel="requestChannel" 
-					url="http://localhost:8080/receiveGateway"
+					url="#{url}"
 					http-method="POST"
 					expected-response-type="java.lang.String"/>
 
+	<beans profile="default">
```

Comment:
I was playing around with a `cloud` profile.

I decided not to pursue it in this PR but didn't see any reason not to show how different URLs can be used via profiles.